src/settings/settings.h

```diff
@@ -23,11 +23,20 @@ namespace settings {
     /// <summary> Zone specific settings. </summary>
     struct ZoneSettings {
 
+        /// <summary> Recycle mode determines a Zone's behavior when it is collected by GC. </summary>
+        enum class RecycleMode {
+            Auto,
+            Manual
+        };
+
         /// <summary> The zone id. </summary>
         std::string id;
 
         /// <summary> The number of zone workers. </summary>
-        uint32_t workers = 2;
+        uint32_t workers = 1;
```

Comment:
>1 [](start = 27, length = 1)

The same concern with that of ts change. 
Why do you want to change this default value?